public/scripts/main.js

```diff
@@ -17,6 +17,24 @@ const FIREFOX_RIGHT_COMMAND_STRING = 'OSRight';
 const CHROME_LEFT_COMMAND_STRING = 'MetaLeft';
 const CHROME_RIGHT_COMMAND_STRING = 'MetaRight';
 
+// Key names to ID
+const keyToId = {
+  ctrl: 'control',
+  control: 'control',
+  command: 'metaleft',
+  cmd: 'metaleft',
+  meta: 'metaleft',
+  fn: 'fnc',
+  alt: 'optionleft',
+  shift: 'shiftleft',
+  esc: 'escape',
+  tab: 'tab',
+  'space bar': 'space',
+  'tilde(~)': 'tilde',
+  'comma(,)': 'comma',
+  'underscore(_)': 'minus'
+};
+
```

Comment:
Hi, I found a few other functions which can use this improvement and not just promptKey. Perhaps for consistency, it would be better to change those too.

Also, if you edit your description to use one of the special keywords, it will automatically link the issue and make it easier for you to get credit for the PR.

Reply:
@apoorva-mk thanks for reviewing and please do create other issue(s) for any similar beneficial refactorings. We'll just leave this one scoped to this particular function.